static/src/javascripts/projects/common/modules/experiments/tests/sign-in-gate-mandatory-long.js

```diff
@@ -0,0 +1,24 @@
+export const signInGateMandatoryLongTestRun = {
+	id: 'SignInGateMandatoryLongTestRun',
+	start: '2022-08-01',
+	expiry: '2022-10-01',
+	author: 'vlbee',
+	description:
+		'Show mandatory sign in gate to global users on 3rd article view of simple article templates, with higher priority over banners and epic.',
+	audience: 0.0001, // todo
```

Comment:
It might be easier to understand these numbers as percentages:

```suggestion
	audience: 1 / 10_000, // todo
```